packages/components/src/index.ts

```diff
@@ -169,9 +169,21 @@ export {
 	ToolbarItem,
 } from './toolbar';
 export {
+	/**
+	 * @deprecated Import `ToolsPanel` instead.
+	 */
 	ToolsPanel as __experimentalToolsPanel,
+	/**
+	 * @deprecated Import `ToolsPanelItem` instead.
+	 */
 	ToolsPanelItem as __experimentalToolsPanelItem,
```

Comment:
This is another component that probably should have its children namespace into it (`ToolsPanel`,`ToolsPanel.Item`, `ToolsPanel.Context`) cc @mirka